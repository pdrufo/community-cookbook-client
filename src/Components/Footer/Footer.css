footer {
  position: absolute;
  background-color: #C2653C;
  border: 1px solid #131B2D;
  color: #323BC2;
  width: 100%;
  height: 25px;
  text-align: center;
}

footer p {
  margin: -2px;
}