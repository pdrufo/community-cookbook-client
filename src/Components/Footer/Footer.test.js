
import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom'
import Footer from './Footer';
import renderer from 'react-test-renderer'


describe('Footer' , () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter><Footer /></BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Footer /></BrowserRouter>)
      .toJSON()
    expect(tree).toMatchSnapshot();
  })
})