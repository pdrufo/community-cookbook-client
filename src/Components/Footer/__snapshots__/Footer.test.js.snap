// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the UI as expected 1`] = `
<footer>
  <p>
     © Peter Rufo 2020
  </p>
</footer>
`;

exports[`add workout renders the UI as expected 1`] = `
<footer>
  <p>
     © Peter Rufo 2020
  </p>
</footer>
`;
